clean-abap/CleanABAP.md

```diff
@@ -3959,6 +3959,16 @@ METHOD do_something.
 ENDMETHOD.
 ```
 
+This is also the case within a statment, as this can easily be seen as a new statement when looking through the code.
```

Comment:
```suggestion
This is also the case within a statement, as this can easily be misunderstood as a new statement when skimming through the code.
```
-> typo fix and more clarity(?)